backend/src/packages/map/map.service.ts

```diff
@@ -41,7 +41,7 @@ class MapService {
   public async getPriceByDistance({
     startAddress,
     endAddress,
-    pricePerKm,
+    pricePerKm, // TODO kilometer
```

Comment:
Not sure what that is for.

Reply:
it was remark to myself, but i refused it later and it dropped out of sight